build_scripts/build_wheel.sh

```diff
@@ -105,9 +126,9 @@ for PYTHON_VERSION in 3.9; do
         fi
         HOST_WHL="theseus_ai-${TAG}${PLUS_CU_TAG}-${CP_STR}-${CP_STR}-manylinux_2_17_x86_64.whl"
     fi
-    
-    sudo docker cp "${DOCKER_NAME}:theseus/dist/theseus-ai-${TAG}.tar.gz" "${DOCKER_DIR}/theseus-ai-${TAG}.tar.gz"
-    sudo docker cp "${DOCKER_NAME}:${DOCKER_WHL}" ${DOCKER_DIR}/${HOST_WHL}
+
+    sudo docker cp "${DOCKER_NAME}:theseus/dist/theseus-ai-${TAG}.tar.gz" "theseus-ai-${TAG}.tar.gz"
```

Comment:
no, I was getting an error because there is already a "cd $DOCKER_DIR" earlier, and it was trying to save into wheel/something/wheel/something, duplicating the path.

Reply:
Ah ok, must be that I was using an absolute path and you were using a relative path?